.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,37 @@
+name: Markdown Lint
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
```

Comment:
This is a repository for templated workflows. Please make this workflow reusable.

Check: https://docs.github.com/en/actions/using-workflows/reusing-workflows

Reply:
Done. Ready for review

Reply:
Create folder docs/ and put Markdown file in there.